exporter/tanzuobservabilityexporter/trace_exporter.go

```diff
@@ -162,12 +162,14 @@ func (e *tracesExporter) shutdown(_ context.Context) error {
 }
 
 func mapToSpanTags(tags map[string]string) []senders.SpanTag {
-	var spanTags []senders.SpanTag
+	spanTags := make([]senders.SpanTag, len(tags))
```

Comment:
Let's keep appends and avoid tracking `i` additionally. It must be the same performance
```suggestion
	spanTags := make([]senders.SpanTag, 0, len(tags))
```